nextroute/solution_format.go

```diff
@@ -130,6 +143,58 @@ func Format(
 	return output
 }
 
+// DefaultStatistics creates default custom statistics for a given solution.
+func DefaultStatistics(solution Solution) CustomStatistics {
```

Comment:
Same comment here about the naming / way we add custom statistics (see above). :relaxed:

Reply:
Yep, see above.